servers/src/test/java/tachyon/worker/block/TieredBlockStoreTestUtils.java

```diff
@@ -71,7 +71,7 @@
    * @param tierCapacity Like {@link #TIER_CAPACITY_BYTES}, should be in the same dimension with
    *        tierPath, each element is the capacity of the corresponding dir in tierPath.
    * @param workerDataFolder When specified it sets up the tachyon.worker.data.folder property.
-   * @throws Exception When error happens during creating temporary folder.
+   * @throws Exception When error happens during creating temporary folder
```

Comment:
Actually, after second-thought, I think this will be too strict, because there might be acronyms or other valid nouns of uppercase